plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/guru/VMwareGuru.java

```diff
@@ -267,6 +267,10 @@ long getClusterId(long vmId) {
             if (destData.getObjectType() == DataObjectType.VOLUME && destStoreTO.getRole() == DataStoreRole.Primary && srcData.getObjectType() == DataObjectType.TEMPLATE
                     && srcStoreTO.getRole() == DataStoreRole.Primary) {
                 needDelegation = false;
+            /* TODO: Uncomment this to disable delegation to SSVM, for copying the template from secondary to primary
```

Comment:
what is the decision to take here @sureshanaparti ? when can we decide? and what do we need for that?